### PR TITLE
Made event post for techtalk with Monica Beckwith

### PR DESCRIPTION
**Pull request by fat1996**
Suggestions on what to add for event description? Or this this enough?

### CODE REVIEW

**Comment by evykassirer** [comment]
please use the abstract Monica emailed us. this should be on all advertising for the talk

File: content/W2017/W17-event-Techtalk-with-Monica-Bechwith.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Title: Techtalk with Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: social
+Slug: tech-talk-w17
```
Comment (annalorimer) [inline content/W2017/W17-event-Techtalk-with-Monica-Bechwith.md:5]: This slug need to be more descriptive.

File: content/W2017/W17-event-Techtalk-with-Monica-Bechwith.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Author: Fatima Taj
+Summary: Come join us at a tech talk with Monica Beckwith!
+
+Come learn about JIT, Runtime Optimizations, and HotSpot garbage collection with Monica Beckwith!
```
Comment (annalorimer) [inline content/W2017/W17-event-Techtalk-with-Monica-Bechwith.md:9]: Use the talk abstract here.

File: content/W2017/W17-event-Techtalk-with-Monica-Bechwith.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+## Event Details ##
+
++ **Who:** Monica Beckwith
```
Comment (annalorimer) [inline content/W2017/W17-event-Techtalk-with-Monica-Bechwith.md:13]: This should detail who is permitted to attend the event, in this case everyone 😊

File: content/W2017/W17-event-Techtalk-with-Monica-Bechwith.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+Title: Techtalk with Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: social
```
Comment (annalorimer) [inline content/W2017/W17-event-Techtalk-with-Monica-Bechwith.md:4]: I think there's a 'talks' tag that should be added.

**Review by annalorimer — CHANGES_REQUESTED** [review]

**Pull request closed without merging**